setup.sh

```diff
@@ -143,8 +143,8 @@ elif [[ "$MODE" == "stable" || ! -v MODE ]]; then
             echo "Installing stable jax, jaxlib, libtpu for NVIDIA gpu"
             pip3 install "jax[cuda12_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
         fi
-        pip3 install "transformer-engine==1.5.0+297459b" \
-          --extra-index-url https://us-python.pkg.dev/gce-ai-infra/maxtext-build-support-packages/simple/
+        export NVTE_FRAMEWORK=jax
+        pip3 install git+https://github.com/NVIDIA/TransformerEngine.git@v1.5
```

Comment:
Have you tested @stable version for stable mode? https://github.com/NVIDIA/TransformerEngine?tab=readme-ov-file#pip suggests using @stable version.